packages/expo/src/__mocks__/Constants-development.ts

```diff
@@ -8,6 +8,7 @@ const Constants = (require as any).requireActual('expo-constants').default;
 const MockConstants = Object.create(Constants);
 MockConstants.manifest = {
   ...Constants.manifest,
+  executionEnvironment: 'standalone',
```

Comment:
somewhat arbitrary choice, i decided standalone was as good as any executionEnvironment to default to in the tests